types/massive/index.d.ts

```diff
@@ -265,29 +427,46 @@ declare namespace massive {
         /** List all the functions and scripts attached to the connected instance. */
         listFunctions(): Promise<any[]>;
 
+        /** List all the non-pk sequences attached to the connected instance. */
+        listSequences(): Promise<any[]>;
+
         /** List all the tables attached to the connected instance. */
         listTables(): Promise<any[]>;
 
         /** List all the views attached to the connected instance. */
         listViews(): Promise<any[]>;
 
         /** Execute a query. */
-        query(query: Select | Insert | Update | Delete | string, params?: any, options?: QueryOptions): Promise<any>;
+        query(
+            query: Select | Insert | Update | Delete | string,
+            params?: string[],
+            options?: Pick<
+                ResultProcessingOptions,
+                "document" | "single" | "stream"
```

Comment:
This is typescripts way of reducing an object type's allowed values. So in this case, from the docs, it looked like the `query` methods options only allowed for `document`, `single`, and `stream` instead of the full `ResultProcessingOptions`. If that's not the case, let me know and I can update it!

Reply:
Nope, you can use all of them (although `build` is nigh-useless since you're already passing raw SQL to `query()`).